content/en/docs/Overview/_index.md

```diff
@@ -7,3 +7,41 @@ description: >
 ---
 
 Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+
+This documentation contains information about both the [parquet-mr](https://github.com/apache/parquet-mr) and [parquet-format](https://github.com/apache/parquet-format) projects. 
+
+
+### parquet-format
+
+The parquet-format project hosts the official specification of the Parquet file format, defining how data is structured and stored. This specification, along with Thrift metadata definitions and other crucial components, is essential for developers to effectively read and write Parquet files. The parquet-format project specifically contains the format specifications needed to understand and properly utilize Parquet files.
+
+As a repository focused on specification, the parquet-format repository does not contain source code. 
+
+
+### parquet-mr
+
+The parquet-mr GitHub repository is part of the Apache Parquet project and specifically focuses on providing Java tools for handling the Parquet file format within the Hadoop ecosystem. Essentially, this repository includes all the necessary Java libraries and modules that allow developers to read and write Parquet files.
+
+The parquet-mr repo contains a reference implementation of the Parquet format. There are a number of other Parquet format implementations, which are listed below. 
```

Comment:
It's "a" reference implementation, it means there are other ones. But I don't see them listed here.

Either list all reference implementations explicitly, or make this "the" reference implementation.

Reply:
If we knew what those references implementations are, I agree it would be valuable to document.

However,  I think there is consensus required before we made such a determination

Thus, for this PR I suggest:
1. Remove the word "reference"
2. File a follow on ticket / discussion in the mailing list to figure out what should be listed as references implementations
```suggestion
The parquet-mr repo contains an implementation of the Parquet format. There are a number of other Parquet format implementations, which are listed below. 
```

Reply:
There have been a lot of conversations about this: https://github.com/apache/parquet-site/pull/53#discussion_r1582882267 (and others on the thread) and I'm inclined to keep this as is. I don't think we need to exhaustively list other the reference reference implementing when there is a list of implementations below.  @gszadovszky has also called this a reference implementation and I think it helps clarify the relationship between the `parquet-format` and `parquet-mr`. I'm more than happy to update this once the community has reached a consensus after the mailing list discussion that @alamb suggested though.

Reply:
If we don't agree on what a reference implementation then we should not list parquet-mr as a reference implementation. The term "reference implementation" has an official connotation and implies a specific status; it certainly should not be assigned lightly.

Reply:
cc @gszadovszky @Fokko @xhochy @wgtmac. I don't want to further block this PR by settling this beforehand, so I'm going to remove the word "reference" and we can add it back if we want to.

Reply:
I started a thread on the mailing list about this topic to see if we can reach consensus: https://lists.apache.org/thread/f9379yx0lf5gtpkgyv922pvowtzy4kmm